app/models/grda_warehouse/shape/county.rb

```diff
@@ -27,10 +28,18 @@ def name
       end
 
       def self.zip_codes
+        ZipCode.joins(<<~SQL)
+          JOIN shape_counties ON (
+            LOWER(shape_counties.namelsad) = shape_zip_codes.county_name_lower
+          )
+        SQL
+      end
+
+      def self.spacial_zip_codes
```

Comment:
probably my original spelling mistake. should be "spatial"

Reply:
I think that was me, I knew it didn't look right.  Thanks!